code-challenges.test.js

```diff
@@ -50,5 +82,22 @@ var testArray2 = [7, 8, 2, 3, 1, 5, 4]
 // Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
 
 
+describe("scream", () => {
+    var testArray1 = [3, 7, 10, 5, 4, 3, 3]
+    var testArray2 = [7, 8, 2, 3, 1, 5, 4]
+
+    it("take in two array arguments", () => {
+        expect(scream(testArray1)).toEqua[3, 7, 10, 5, 4, 8, 2, 1];
+    })
+})
 
 // b) Create the function that makes the test pass.
+
+let scream1 = [3, 7, 10, 5, 4, 3, 3]
+let scream2 = [7, 8, 2, 3, 1, 5, 4]
+let scream3 = scream1.concat(scream2);
+
+const holler = (arr) => {
+    return scream3 = [...new Set([...scream1, ...scream2])];
+}
+// Please advise on this set up.
```

Comment:
I'd love to hear more about your thought process in your pseudocode. It will make it easier for me to advise you. `new Set` is a very cool approach. But remember the fundamentals of functions: functions are encapsulated. Meaning the only data they have access to is what is passed in via the argument.

Also need to make sure the function is matching the test. The test is calling a function named `scream` so that is what your function should be named.
```
const scream = (arr1, arr2) => {
    return [...new Set([...arr1, ...arr2])]
}
```